docs/tutorials/plotting/pairplot.ipynb

```diff
@@ -119,7 +126,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.0"
+   "version": "3.8.5"
```

Comment:
will this cause issues with sphinx?

Reply:
@loftusa there was some weird thing were it was not rendering in sphinx correctly one time and that is cause i had ran the notebook locally and had an import error because i was not in my correct env. TLDR I am unsure because this tutorial was not responsible and I made no code changes to it outside of clearing output cells before pushing.